framework/src/main/java/org/checkerframework/framework/type/ElementAnnotationApplier.java

```diff
@@ -188,7 +223,11 @@ public Void visitTypeVariable(AnnotatedTypeVariable type, AnnotatedTypeFactory f
             if (type.getAnnotations().isEmpty()
                     && type.getUpperBound().getAnnotations().isEmpty()
                     && tpelt.getEnclosingElement().getKind() != ElementKind.TYPE_PARAMETER) {
-                ElementAnnotationApplier.applyInternal(type, tpelt, factory);
+                try {
+                    ElementAnnotationApplier.applyInternal(type, tpelt, factory);
+                } catch (UnexpectedAnnotationLocationException e) {
+                    // Error was already reported by #apply.
```

Comment:
This still isn't clear to me... Which invocation of `#apply`? Are we processing the same annotations twice and this is the second iteration and therefore we don't need to issue errors again?
Can you add a short comment when this earlier invocation of `#apply` happened?